lib/uploads/create.js

```diff
@@ -1,419 +0,0 @@
-/*
```

Comment:
Is this some MPU stuff that's being removed?

Reply:
i don't know the specifics of this - i just know that the file wasn't required by anything but matched the same logic around durability level (that's how i found this file)